tests/vcd_role.py

```diff
@@ -26,5 +41,13 @@ def test_get_rights(self):
         rights = role.list_rights()
         assert len(rights) > 0
 
+    def test_create_role(self):
+        logged_in_org = self.client.get_org()
+        org = Org(self.client, resource=logged_in_org)
+        role_name = self.config['vcd']['role_name']
+        org.create_role(role_name, 'test description', ('Disk: View Properties',))
+        role_record = org.get_role(role_name)
+        assert self.config['vcd']['role_name'] == role_record.get('name')
```

Comment:
Please make sure you clean up the role after the tests are done.

Reply:
Yes, I will make a not to modify this test once I implement the command to delete a role.